frontend/src/HomePage/HomePage.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import HomePage from "./HomePage";
+
+test("renders learn react link", () => {
+  const { getByText } = render(<HomePage />);
+  getByText("BEAP ENGINE");
+  getByText("Accurately Process Your Fitness Data");
+  getByText("How To Contribute Data");
+  getByText(
+    "BEAP Engine is a research project developed by Dr. Daniel Fuller",
+    { exact: false },
```

Comment:
also works to do getByText((content, element) => content.startsWith('BEAP Engine is a research ')